clients/client/test/retry_test.js

```diff
@@ -1,260 +1,265 @@
-const taskcluster = require('../');
+// const taskcluster = require('../');
 const assert = require('assert');
-const SchemaSet = require('taskcluster-lib-validate');
-const { APIBuilder } = require('taskcluster-lib-api');
-const testing = require('taskcluster-lib-testing');
-const { App } = require('taskcluster-lib-app');
-const { monitorManager, monitor } = require('./monitor');
+// const testing = require('./helper');
 const retry = require('../src/retry');
 
-const rootUrl = `http://localhost:60526`;
+// IMPLICIT DEPENDENCIES ...
+// const { APIBuilder } = require('taskcluster-lib-api');
+// const { App } = require('taskcluster-lib-app');
+// const SchemaSet = require('taskcluster-lib-validate');
+// const { monitorManager, monitor } = require('./monitor');
 
-suite(testing.suiteName(), function() {
-  let proxier;
+// const rootUrl = `http://localhost:60526`;
 
-  // Construct API
-  let builder = new APIBuilder({
-    title: 'Retry API',
-    description: 'API that sometimes works by retrying things',
-    serviceName: 'retrytest',
-    apiVersion: 'v1',
-  });
+// This depends on the library packages from the outside world..
+// this test needs to be included inside the main app instead
 
-  let getInternalErrorCount = 0;
-  builder.declare({
-    method: 'get',
-    route: '/internal-error',
-    name: 'getInternalError',
-    title: 'Test End-Point',
-    scopes: 'test:internal-error',
-    category: 'Taskcluster Client',
-    description: 'Place we can call to test something',
-  }, function(req, res) {
-    getInternalErrorCount += 1;
-    res
-      .status(500)
-      .json({
-        message: 'Internal error of sorts',
-      });
-  });
+// suite(testing.suiteName(), function() {
```

Comment:
as this whole test depends on "internal" libraries, it shouldn't be here.
figuring out where to put it